app/scripts/lib/experiments/open-gmail.js

```diff
@@ -0,0 +1,62 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'underscore',
+  'lib/experiments/base'
+], function (_, BaseExperiment) {
+  'use strict';
+
+  var EXPERIMENT_NAME = 'openGmail';
+  var GMAIL_PATH = 'https://mail.google.com/mail/u/?authuser=';
+
+  function OpenGmailExperiment() {
+    // constructor
+  }
+
+  _.extend(OpenGmailExperiment.prototype, new BaseExperiment(), {
+    initialize: function (options) {
+      if (! options || ! options.account) {
+        return false;
+      }
+
+      options.extraAbleOptions = {
+        email: options.account.get('email')
+      };
+
+      return this.start(EXPERIMENT_NAME, options);
+    },
+    getContext: function () {
+      return {
+        isOpenGmailButtonVisible: this.isGroup('treatment')
+      };
+    },
+    performAction: function (action) {
+      if (action === 'openGmailTab') {
+        this.saveState('clicked');
```

Comment:
Saves the state of the experiment that the user "clicked" on the open gmail button